src/Domain/FunctionsOfTime/QuaternionHelpers.cpp

```diff
@@ -35,5 +36,8 @@ void normalize_quaternion(
   *input /= abs(*input);
 }
 
-template void normalize_quaternion(
-    gsl::not_null<boost::math::quaternion<double>*> input) noexcept;
+template <>
+void normalize_quaternion(
```

Comment:
Sorry, I meant make this a normal function, not a template specialization, and remove the generic template because it is impossible to call.